<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CustomerAdministration</class>
 <widget class="QWidget" name="CustomerAdministration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>581</width>
    <height>482</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Kundenverwaltung</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Suchen nach:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="inputSearch"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Durchsuche:</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCheckBox" name="checkFilterCustomerName">
       <property name="text">
        <string>Kundennamen</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkFilterAddress">
       <property name="text">
        <string>Adresse</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkFilterPLZ">
       <property name="text">
        <string>PLZ</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkFilterCity">
       <property name="text">
        <string>Ort</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushSearch">
       <property name="text">
        <string>Suchen</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTreeView" name="treeViewCustomers"/>
   </item>
   <item>
    <widget class="QPushButton" name="pushNewCustomer">
     <property name="text">
      <string>Neuer Kunde</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
